### PR TITLE
Add New York

### PR DESCRIPTION
**Pull request by systemcatch**
This gets production for the entire day.  No exchanges until it's decided how best to implement them.

ref #898

### CODE REVIEW

File: parsers/US_NY.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+import arrow
+from collections import defaultdict
+from operator import itemgetter
+import pandas as pd
+
+
+ny = arrow.now('America/New_York')
+#Mix csv requires zero padded day.
+ny_date = str(ny.year) + str(ny.month) + (str(ny.day) if ny.day > 9 else str(ny.day).zfill(2))
```
Comment (jarek) [inline parsers/US_NY.py]: You can also do `datetime.datetime.strftime(ny.datetime, '%Y%m%d')` - %d is zero-padded ([ref](https://docs.python.org/2/library/datetime.html#strftime-strptime-behavior))

or use the arrow format function - `ny_date = ny.format('YYYYMMDD')` ([ref](http://arrow.readthedocs.io/en/latest/#arrow.arrow.Arrow.format))

**Comment by corradio** [comment]
Awesome @systemcatch ! Can you use the script that updates the mockserver and post screenshot?

File: config/zones.json [hunk 2]
```diff
@@ -1477,6 +1477,16 @@
   "US-HI": {
     "flag_file_name": "us.png"
   },
+  "US-NY": {
+    "parsers": {
+      "production": "US_NY.fetch_production"
+    },
+    "flag_file_name": "us.png",
+    "contributors": [
+      "https://github.com/corradio",
```
Comment (corradio) [inline config/zones.json]: No need to add my name :)

File: parsers/US_NY.py (hunk 1 above)
Comment (corradio) [inline parsers/US_NY.py]: This should be defined inside a method (else it will stay constant throughout time)

Comment (systemcatch) [reply parsers/US_NY.py]: ah that's much better than my hack

File: config/zones.json (hunk 2 above)
Comment (systemcatch) [reply config/zones.json]: Sure?  You split the US on the map.

Comment (corradio) [reply config/zones.json]: yeah let's keep people who worked exclusively on the parser

**Comment by systemcatch** [comment]
new_york

**Comment by systemcatch** [comment]
I'm not sure why CircleCI failed on the Fix storage error commit.  Formatting issue?

**Comment by corradio** [comment]
I think it is a bug in `npm`. If tests pass now, then nothing to worry about.

**Comment by HansHyde** [comment]
@systemcatch noted your division efforts with Chile

**Pull request merged**